experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -756,16 +903,17 @@ export class HttpInstrumentation extends InstrumentationBase<HttpInstrumentation
     request: http.IncomingMessage,
     response: http.ServerResponse,
     span: Span,
-    metricAttributes: Attributes,
+    oldMetriceAttributes: Attributes,
+    stableMetriceAttributes: Attributes,
     startTime: HrTime
   ) {
     const attributes = getIncomingRequestAttributesOnResponse(
       request,
       response,
       this._semconvStability
     );
-    metricAttributes = Object.assign(
-      metricAttributes,
+    oldMetriceAttributes = Object.assign(
+      oldMetriceAttributes,
       getIncomingRequestMetricAttributesOnResponse(attributes)
     );
```

Comment:
IIUC `http.route` is not attached to the new metrics. Is this intentional? :thinking: 

Users usually ask for this attribute as it ends up being the most important attribute for them (not being able to split by it means there's no way to distinguish between their different endpoints)

Reply:
I left it off because the http module has no concept of route. Forgot that it coordinates with framework instrumentations. I'll add it.